flamedisx/xenon/x1t_sr1.py

```diff
@@ -95,7 +95,7 @@ def interpolate_tf(sig_tf, fmap, domain):
             x_ref_min=domain[0], x_ref_max=domain[1], 
             y_ref=fmap, fill_value='constant_extension')
 
-def cal_bias_tf(sig, fmap, domain_def, pivot_pt):
+def interpolate_reconstruction_bias(sig, fmap, domain_def, pivot_pt):
```

Comment:
Minor comment but it would be better to call it `calculate_reconstruction_bias` to reflect its functionality. It carries out some computation instead of just interpolating from the bias maps.

Reply:
It's several interpolations in sequence: first you interpolate the different bias maps at the observed signals, then you interpolate the results for the chosen pivot. But happy to change it back :-)

Reply:
hahah yes i'm nitpicking but yes thanks!

there are 2 lines of interpolation, and then 2 lines of calculations from the interpolated results, not quite interpolating the results itself. https://github.com/FlamTeam/flamedisx/blob/a709e1747f0044e8cab95fb1fe5831b37586348d/flamedisx/xenon/x1t_sr1.py#L122

but yes. thanks!